### PR TITLE
[fix] Netex - Skip line without network

### PR DESCRIPTION
**Pull request by ArnaudOggy**
Structure Line/RepresentedByGroupRef/@ref can not exist. It is not an obligatory field, and according to specifications the line should be skipped.

### CODE REVIEW

**Comment by woshilapin** [comment]
I thought we said we wouldn't fix this one? If we decide to fix it, why not also the `OperatorRef` and all the other missing fields?

File: src/netex_idf/lines.rs [hunk 1]
```diff
@@ -116,9 +116,9 @@ fn load_netex_lines(
                     .map(Element::text)
                     .ok();
                 let private_code = line.only_child("PrivateCode").map(Element::text);
-                let network_id: String = line
-                    .try_only_child("RepresentedByGroupRef")?
-                    .try_attribute("ref")?;
+                let network_id: String = skip_fail!(line
```
Comment (woshilapin) [inline src/netex_idf/lines.rs]: How about adding a test for that one?

Comment (ArnaudOggy) [reply src/netex_idf/lines.rs]: I thought about doing it yesterday. I just wrote that before leaving.

**Comment by ArnaudOggy** [comment]
> Are the other fields optional too? Like `OperatorRef`, `PublicCode`, `ShortName` or `TransportMode`?
OperatorRef, PublicCode and TransportMode are declared mandatory in their specifications.
ShortName is depreciated, but we use PublicCode primarily.

**Pull request merged**